### DISCUSSION TITLE
Sandbox Service failed to create Sandbox

### DISCUSSION THREAD

**Discussion by ghost** [openkfw/TruBudget#1281] [Ideas]
I am getting an error msg. Sandbox Service failed to create Sandbox

**Comment by Stezido** [comment]
Hi,

with the major update of TruBudget (2.0) TaaS has accepted 2.0 version user input, but our service is nopt ready for 2.0 yet.
Unfortunately the default version when creating a sandbox was the latest so a 2.x version which leads to this error.

We released a hotfix where it's not possible anymore to choose from 2.x versions.
Spaces in sandbox names are also forbidden now, since they also have been leading to sandbox errors.
